js/leaflet-sidebar.js

```diff
@@ -156,6 +156,11 @@ L.Control.Sidebar = L.Control.extend(/** @lends L.Control.Sidebar.prototype */ {
             if (L.DomUtil.hasClass(child, 'active'))
                 L.DomUtil.removeClass(child, 'active');
         }
+        for (i = this._panes.length - 1; i >= 0; i--) {
+            var child = this._panes[i];
+            if (L.DomUtil.hasClass(child, 'active'))
+                L.DomUtil.removeClass(child, 'active');
+        }
```

Comment:
I admit I haven't tested this yet, but won't this flicker during the collapsing animation? IIRC I kept the `active` class on the content so that the content is still visible while the sidebar is collapsing. If you remove the `active` class before the sidebar is collapsed the content probably won't be visible anymore during that animation.